guide/src/class/protocols.md

```diff
@@ -410,3 +410,5 @@ impl ClassWithGCSupport {
 [`IterNextOutput`]: {{#PYO3_DOCS_URL}}/pyo3/class/iter/enum.IterNextOutput.html
 [`PySequence`]: {{#PYO3_DOCS_URL}}/pyo3/types/struct.PySequence.html
 [`CompareOp::matches`]: {{#PYO3_DOCS_URL}}/pyo3/pyclass/enum.CompareOp.html#method.matches
```

Comment:
I _think_ it's required that these markdown links go at the bottom of the document, i.e. the note needs to go above them?

Reply:
moved just in case